templates/zerver/apps.html

```diff
@@ -86,10 +86,17 @@ <h3 class="center">{% trans %}Ok, I take it back, I'm sorry, please don't go.{%
                             <div class="wrapper">
                                 <h2>MacOS</h2>
                                 <div class="instructions">
-                                    <p class="center">You love your Mac. And you love Zulip. So what could be better than a Zulip app for Mac? Enjoy notifications for messages and PMs in your dock whether you're in Sublime, emacs, or Photoshop.</p>
+                                    <p class="center">
+                                        {% trans %}You love your Mac. And you
+                                        love Zulip. So what could be better
+                                        than a Zulip app for Mac? Enjoy
+                                        notifications for messages and PMs in
+                                        your dock whether you're in Sublime,
+                                        emacs, or Photoshop.{% endtrans %}
+                                    </p>
                                 </div>
                                 <a href="https://zulip.org/dist/apps/mac/Zulip-latest.dmg" download>
-                                    <button class="button grey-transparent" type="button" name="button">Download Now</button>
+                                    <button class="button grey-transparent" type="button" name="button">{{ _('Download now') }}</button>
```

Comment:
:+1: for these capitalization changes -- I took a quick look at a couple of other sites, and this capitalization style seems to be more current.